build/KoreBuild.ps1

```diff
@@ -91,13 +93,9 @@ if (!(Test-Path "$koreBuildFolder\Sake"))
     Invoke-WebRequest "https://dist.nuget.org/win-x86-commandline/v3.5.0-beta2/NuGet.exe" -OutFile "$koreBuildFolder/nuget.exe"
 }
 
-$makeFilePath = "makefile.shade"
-if (!(Test-Path $makeFilePath))
-{
-    $makeFilePath = "$koreBuildFolder\shade\makefile.shade"
-}
-
-Write-Host "Using makefile: $makeFilePath"
-
 $env:KOREBUILD_FOLDER=$koreBuildFolder
-&"$koreBuildFolder\Sake\0.2.2\tools\Sake.exe" -I $koreBuildFolder\shade -f $makeFilePath @args
+$makeFileProj = "$koreBuildFolder/targets/makefile.proj"
+if ($allparams) {
+    $targets = '/t:' + $allparams.Replace(' ', ';')
+}
+dotnet msbuild $makeFileProj /p:KoreBuildDirectory="$koreBuildFolder\" /p:RepositoryRoot="$repoFolder\" $targets
```

Comment:
$targets needs to be surrounded with double quotes so multiple targets doesn't terminate the line.

Suggested adding these parameters:
/maxcpucount /fileLogger /fileloggerparameters:LogFile=artifacts/msbuild.log /detailedsummary

Also, consider making an rsp file (artifacts/msbuild.rsp) and executing "dotnet msbuild @artifacts/msbuild.rsp" instead.

Nit: add newline at end of file.